citesphere/src/test/java/edu/asu/diging/citesphere/core/service/oauth/impl/OAuthClientManagerTest.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.diging.citesphere.core.service.oauth.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import edu.asu.diging.citesphere.core.repository.oauth.OAuthClientRepository;
+
+public class OAuthClientManagerTest {
+    
+    @Mock
+    private OAuthClientRepository clientRepo;
+    
+    @Mock
+    private BCryptPasswordEncoder bCryptPasswordEncoder;
+    
+    private int accessTokenValidity = 3600;
+    
+    @InjectMocks
+    private OAuthClientManager managerToTest = new OAuthClientManager(clientRepo, bCryptPasswordEncoder, accessTokenValidity);
```

Comment:
you do not need to instantiate this, Mockito should be able to do that for you, have you tried it?

Reply:
I tried but it failed with following error:

Cannot instantiate @InjectMocks field named 'managerToTest'.
You haven't provided the instance at field declaration so I tried to construct the instance.
However, I failed because: the type 'OAuthClientManager' has no default constructor